src/base-component/AuthLargeLayout.jsx

```diff
@@ -30,7 +32,7 @@ const AuthLargeLayout = ({ intl, username }) => (
     </div>
     <div className="col-md-2 bg-white p-0">
       <svg className="m1-n1 w-100 h-100 large-screen-svg-light" preserveAspectRatio="xMaxYMin meet">
-        <g transform="skewX(171.6)">
+        <g transform={`skewX(${isRtl(getLocale()) ? '-' : ''}171.6)`}>
```

Comment:
@brian 
1. I agree with you about improving clarity by putting the number inside the ternary expression `_ ? -num : num`.
2. `_ & _` won't work inside a string template as it gives a `'false'` string instead of an empty string ''

So I have made the following updates:
- adopted option 1 above.
- edited a failing test files and tests are passing.
- run eslint and fixed any errors (unlike other mfe repos, this one doesn't have a pre-commit hook to run eslint so the errors just slept by)